databricks/koalas/tests/test_dataframe.py

```diff
@@ -426,3 +426,54 @@ def test_to_pandas(self):
         pdf = self.pdf
         self.assert_eq(kdf.toPandas(), pdf)
         self.assert_eq(kdf.to_pandas(), pdf)
+
+    def test_isin(self):
+        df = pd.DataFrame({'a': [4, 2, 3, 4, 8, 6],
+                           'b': [1, 2, 9, 4, 2, 4],
+                           'c': ["one", "three", "six", "seven", "one", "5"]},
+                          index=[10, 20, 30, 40, 50, 60])
+
+        kdf = koalas.from_pandas(df)
+        self.assert_eq(kdf.isin([4, 'six']), df.isin([4, 'six']))
+        self.assert_eq(kdf.isin({"a": [2, 8], "c": ['three', "one"]}),
+                       df.isin({"a": [2, 8], "c": ['three', "one"]}))
+
+        msg = "'DataFrame' object has no attribute {'e'}"
+        with self.assertRaisesRegex(AttributeError, msg):
+            kdf.isin({"e": [5, 7], "a": [1, 6]})
+
+        msg = "Dataframe and Series are not supported"
+        with self.assertRaisesRegex(NotImplementedError, msg):
+            kdf.isin(df)
+
+        msg = "Values should be iterable, Series, DataFrame or dict."
+        with self.assertRaisesRegex(TypeError, msg):
+            kdf.isin(1)
+
+        # Test list sanitizer
+        msg = "List contains unsupported type <class 'pandas.core.frame.DataFrame'>"
+        with self.assertRaisesRegex(TypeError, msg):
+            kdf.isin([df, df])
+
+        msg = "List contains unsupported type <class 'pandas.core.frame.DataFrame'>"
+        with self.assertRaisesRegex(TypeError, msg):
+            kdf.isin({"a": [5, df]})
+
+    def test_isin_series(self):
```

Comment:
I'd create test_series.py file and move this there.

Reply:
So, we move also these methods: `test_Series()`, `test_rename_series()`, `test_rename_series_method()` to `test_series.py` ?

Reply:
You can create a new file and just put this test case there, and create a new pull request in the future to move the rest if you want. In a way that's better because the commits are more atomic and self-contained.

Reply:
Sounds good!